src/helpers/bodyTrap.js

```diff
@@ -53,7 +53,9 @@ function bodyTrap(eventType, openInstances) {
 
   instances = openInstances;
 
-  if (instances.length > 0) {
+  const trapFocus = instances.find(({ props }) => !props.disableFocusTrap);
+
+  if (trapFocus && instances.length > 0) {
```

Comment:
What happens if we open one modal with focus trap disabled and one that's enabled?

Reply:
Then a focus trap will exist. I thought that is the best way to do it as it gives existing functionality priority over the added functionality. I could add something in the readme to let the user know?

Reply:
I added a comment in the readme about this